src/index.css

```diff
@@ -1,13 +1,52 @@
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+   background: #F2F2F2;
 }
 
 code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
+  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.navbar {
+    padding: 10px 20px;
+    display: flex;
+    align-items: center;
+    border: 1px solid #DDE2FF;
+    background:  #FFFFFF;
+}
+
+.logo {
+    display: flex;
+    align-items: center;
+    margin-right: 20px;
+    gap: 10px;
+}
+
+.logo img {
+    width: 34px;
+    height: 34px;
+    top: 19px;
+    left: 150px;
+    padding: 0.015339844860136509px 0px 0.0000028675422072410583px 0px;
```

Comment:
Use rem instead of px and use it in a constant.

Reply:
Is there any reason for using padding value with 18 decimal points ?